styles/styles.css

```diff
@@ -0,0 +1,222 @@
+* {
```

Comment:
Please resolve linting errors

Reply:
fixed.
Mistake for class .user-info in progress because associated with changes in JS

Reply:
Update. user-info mistake fixed too

Reply:
please don't disable linting but rather fix the problem
/* stylelint-disable no-missing-end-of-source-newline */